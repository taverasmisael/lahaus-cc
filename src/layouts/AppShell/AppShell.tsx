import Header from '@/components/Header'
import type { NextPage } from 'next'
import Head from 'next/head'

const DEFAULT_DESCRIPTION =
  '¿Buscas una inmobiliaria? Tenemos una buena noticia, somos mucho más. La tecnología y oferta de finca raíz harán más fácil la compra de tu vivienda nueva."'

const AppShell: NextPage<AppShellProps> = ({
  children,
  description,
  title,
}) => {
  return (
    <>
      <Head>
        <title>{title || 'Inicio'} - La haus</title>
        <meta name="description" content={description || DEFAULT_DESCRIPTION} />
        <link rel="icon" href="/favicon.ico" />
      </Head>
      <Header hasNotifications />
      {children}
    </>
  )
}

interface AppShellProps {
  description?: string
  title?: string
}

export default AppShell
