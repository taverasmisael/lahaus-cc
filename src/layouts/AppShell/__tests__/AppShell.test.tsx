import { render } from '@testing-library/react'

import AppShell from '../AppShell'

describe('layouts/AppShell', () => {
  test('it renders without any extra props', () => {
    const { container } = render(<AppShell />)
    expect(container.firstChild).toMatchSnapshot()
  })
  test('it renders with children', () => {
    const { container } = render(
      <AppShell>
        <p>Hello World</p>
      </AppShell>
    )
    expect(container.firstChild).toMatchSnapshot()
  })
})
