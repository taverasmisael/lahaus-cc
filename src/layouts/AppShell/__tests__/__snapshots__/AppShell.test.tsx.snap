// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layouts/AppShell it renders with children 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="center"
    >
      <div
        class="content"
      >
        <div
          class="flex items-center"
        >
          <div
            class="menuIconContainer"
          >
            <svg
              class="menuIcon"
              fill="none"
              height="32"
              title="Menu"
              width="32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M22 25H2v2h20v-2ZM30 15H2v2h28v-2ZM30 5H2v2h28V5Z"
                fill="#5B6866"
              />
            </svg>
            <span
              class="rounded-full w-3 h-3 absolute top-0 right-0 bg-accent notificationBadge"
              data-testid="notification-badge"
            />
          </div>
          <svg
            class="logo hidden md:inline-block"
            fill="none"
            height="25"
            title="La Haus logo"
            width="125"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M13.046.678v9.598a.69.69 0 0 1-.696.68h-1.393a.69.69 0 0 1-.696-.68V3.754c0-.487-.506-.82-.965-.634L.436 6.675A.69.69 0 0 0 0 7.309V22.98c0 .378.316.68.696.68h8.86a.69.69 0 0 0 .697-.68v-5.99a.69.69 0 0 1 .696-.68h1.393a.69.69 0 0 1 .696.68v5.99c0 .378.316.68.696.68h8.86a.69.69 0 0 0 .697-.68V4.24a.69.69 0 0 0-.435-.634L13.995.052a.69.69 0 0 0-.95.626Z"
              fill="#3ECFAF"
            />
            <path
              d="M47.365 20.677v2.403a.557.557 0 0 1-.562.549H35.965a.557.557 0 0 1-.562-.55V6.273c0-.302.253-.549.562-.549h2.753c.308 0 .561.247.561.549V20.12h7.524a.564.564 0 0 1 .562.556ZM62.364 20.841v2.303a.49.49 0 0 1-.498.487h-1.337c-1.472 0-2.27-.48-2.508-1.592-.024-.108-.174-.139-.238-.046-.712 1.167-2.183 1.916-4.192 1.916-2.935 0-4.795-1.584-4.795-4.041 0-2.504 1.939-4.042 5.657-4.042h3.038v-.386c0-1.43-.736-2.095-2.279-2.095a4.769 4.769 0 0 0-3.093 1.128c-.206.17-.514.178-.704-.015l-1.48-1.522a.473.473 0 0 1-.007-.665c1.305-1.283 3.306-2.04 5.577-2.04 3.592 0 5.578 1.893 5.578 5.062v4.984c0 .433.134.564.498.564h.783Zm-4.873-2.303v-.41h-2.666c-1.654 0-2.358.51-2.358 1.539 0 .996.68 1.56 2.017 1.56 1.828 0 3.007-1.073 3.007-2.689ZM80.3 6.272V23.08a.557.557 0 0 1-.561.549h-2.73a.557.557 0 0 1-.561-.549v-6.87h-7.556a.557.557 0 0 0-.561.549v6.321a.557.557 0 0 1-.562.549h-2.753a.557.557 0 0 1-.562-.549V6.272c0-.301.253-.548.562-.548h2.753c.308 0 .562.247.562.548v6.46h7.555a.557.557 0 0 0 .562-.548V6.272c0-.301.253-.548.561-.548h2.73c.316 0 .561.247.561.548ZM110.541 11.081v12.055c0 .27-.23.495-.507.495h-2.626a.504.504 0 0 1-.506-.495v-1.421c-.736 1.383-2.334 2.28-4.296 2.28-2.8 0-4.375-1.763-4.375-4.862v-8.052c0-.27.23-.495.506-.495h2.706c.277 0 .506.224.506.495v7.102c0 1.684.554 2.504 2.097 2.504 1.701 0 2.777-.974 2.777-2.736v-6.87c0-.27.229-.495.506-.495h2.705a.5.5 0 0 1 .507.495ZM112.956 21.297l1.179-1.553a.548.548 0 0 1 .768-.108c1.06.819 2.531 1.337 3.987 1.337 1.313 0 2.017-.433 2.017-1.098 0-.564-.474-.92-1.598-1.174l-2.223-.51c-2.8-.665-3.9-1.948-3.9-3.733 0-2.48 2.096-4.22 5.633-4.22 2.12 0 3.987.627 5.221 1.647a.533.533 0 0 1 .095.734l-1.202 1.6a.557.557 0 0 1-.752.116c-1.139-.735-2.294-1.052-3.41-1.052-1.281 0-1.883.41-1.883 1.052 0 .463.396.795 1.575 1.05l2.302.534c2.856.664 3.876 1.947 3.876 3.763 0 2.635-2.12 4.32-5.759 4.32-2.421 0-4.525-.742-5.862-2.002a.509.509 0 0 1-.064-.703ZM96.327 20.841v2.303a.49.49 0 0 1-.498.487h-1.337c-1.472 0-2.27-.48-2.508-1.592-.024-.108-.174-.139-.238-.046-.712 1.167-2.183 1.916-4.193 1.916-2.935 0-4.794-1.584-4.794-4.041 0-2.504 1.939-4.042 5.657-4.042h3.038v-.386c0-1.43-.736-2.095-2.279-2.095a4.769 4.769 0 0 0-3.093 1.128c-.206.17-.514.178-.704-.015l-1.48-1.522a.473.473 0 0 1-.007-.665c1.305-1.283 3.306-2.04 5.577-2.04 3.592 0 5.577 1.893 5.577 5.062v4.984c0 .433.135.564.499.564h.783Zm-4.873-2.303v-.41H88.78c-1.654 0-2.358.51-2.358 1.539 0 .996.68 1.56 2.017 1.56 1.836 0 3.015-1.073 3.015-2.689Z"
              fill="#012D26"
            />
          </svg>
          <svg
            class="logo md:hidden"
            fill="none"
            height="32"
            title="La Haus logo"
            width="31"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M17.11.917v12.981c0 .512-.416.92-.914.92H14.37a.918.918 0 0 1-.913-.92V5.077a.919.919 0 0 0-1.266-.857L.571 9.028A.934.934 0 0 0 0 9.885V31.08c0 .512.415.92.913.92h11.62a.918.918 0 0 0 .914-.92v-8.1c0-.512.415-.92.913-.92h1.826c.508 0 .913.419.913.92v8.1c0 .512.415.92.913.92h11.62a.918.918 0 0 0 .913-.92V5.735a.934.934 0 0 0-.57-.857L18.355.071c-.582-.251-1.246.198-1.246.846Z"
              fill="#3ECFAF"
            />
          </svg>
        </div>
        <div
          class="backdrop"
        />
        <button
          aria-label="Cerrar Menú"
          class="close hidden"
          type="button"
        >
          <span
            aria-hidden="true"
            class="closeIcon"
          >
            ×
          </span>
        </button>
        <div
          class="transform nav"
        >
          <svg
            class="logo"
            fill="none"
            height="25"
            title="La haus logo"
            width="125"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M13.046.678v9.598a.69.69 0 0 1-.696.68h-1.393a.69.69 0 0 1-.696-.68V3.754c0-.487-.506-.82-.965-.634L.436 6.675A.69.69 0 0 0 0 7.309V22.98c0 .378.316.68.696.68h8.86a.69.69 0 0 0 .697-.68v-5.99a.69.69 0 0 1 .696-.68h1.393a.69.69 0 0 1 .696.68v5.99c0 .378.316.68.696.68h8.86a.69.69 0 0 0 .697-.68V4.24a.69.69 0 0 0-.435-.634L13.995.052a.69.69 0 0 0-.95.626Z"
              fill="#3ECFAF"
            />
            <path
              d="M47.365 20.677v2.403a.557.557 0 0 1-.562.549H35.965a.557.557 0 0 1-.562-.55V6.273c0-.302.253-.549.562-.549h2.753c.308 0 .561.247.561.549V20.12h7.524a.564.564 0 0 1 .562.556ZM62.364 20.841v2.303a.49.49 0 0 1-.498.487h-1.337c-1.472 0-2.27-.48-2.508-1.592-.024-.108-.174-.139-.238-.046-.712 1.167-2.183 1.916-4.192 1.916-2.935 0-4.795-1.584-4.795-4.041 0-2.504 1.939-4.042 5.657-4.042h3.038v-.386c0-1.43-.736-2.095-2.279-2.095a4.769 4.769 0 0 0-3.093 1.128c-.206.17-.514.178-.704-.015l-1.48-1.522a.473.473 0 0 1-.007-.665c1.305-1.283 3.306-2.04 5.577-2.04 3.592 0 5.578 1.893 5.578 5.062v4.984c0 .433.134.564.498.564h.783Zm-4.873-2.303v-.41h-2.666c-1.654 0-2.358.51-2.358 1.539 0 .996.68 1.56 2.017 1.56 1.828 0 3.007-1.073 3.007-2.689ZM80.3 6.272V23.08a.557.557 0 0 1-.561.549h-2.73a.557.557 0 0 1-.561-.549v-6.87h-7.556a.557.557 0 0 0-.561.549v6.321a.557.557 0 0 1-.562.549h-2.753a.557.557 0 0 1-.562-.549V6.272c0-.301.253-.548.562-.548h2.753c.308 0 .562.247.562.548v6.46h7.555a.557.557 0 0 0 .562-.548V6.272c0-.301.253-.548.561-.548h2.73c.316 0 .561.247.561.548ZM110.541 11.081v12.055c0 .27-.23.495-.507.495h-2.626a.504.504 0 0 1-.506-.495v-1.421c-.736 1.383-2.334 2.28-4.296 2.28-2.8 0-4.375-1.763-4.375-4.862v-8.052c0-.27.23-.495.506-.495h2.706c.277 0 .506.224.506.495v7.102c0 1.684.554 2.504 2.097 2.504 1.701 0 2.777-.974 2.777-2.736v-6.87c0-.27.229-.495.506-.495h2.705a.5.5 0 0 1 .507.495ZM112.956 21.297l1.179-1.553a.548.548 0 0 1 .768-.108c1.06.819 2.531 1.337 3.987 1.337 1.313 0 2.017-.433 2.017-1.098 0-.564-.474-.92-1.598-1.174l-2.223-.51c-2.8-.665-3.9-1.948-3.9-3.733 0-2.48 2.096-4.22 5.633-4.22 2.12 0 3.987.627 5.221 1.647a.533.533 0 0 1 .095.734l-1.202 1.6a.557.557 0 0 1-.752.116c-1.139-.735-2.294-1.052-3.41-1.052-1.281 0-1.883.41-1.883 1.052 0 .463.396.795 1.575 1.05l2.302.534c2.856.664 3.876 1.947 3.876 3.763 0 2.635-2.12 4.32-5.759 4.32-2.421 0-4.525-.742-5.862-2.002a.509.509 0 0 1-.064-.703ZM96.327 20.841v2.303a.49.49 0 0 1-.498.487h-1.337c-1.472 0-2.27-.48-2.508-1.592-.024-.108-.174-.139-.238-.046-.712 1.167-2.183 1.916-4.193 1.916-2.935 0-4.794-1.584-4.794-4.041 0-2.504 1.939-4.042 5.657-4.042h3.038v-.386c0-1.43-.736-2.095-2.279-2.095a4.769 4.769 0 0 0-3.093 1.128c-.206.17-.514.178-.704-.015l-1.48-1.522a.473.473 0 0 1-.007-.665c1.305-1.283 3.306-2.04 5.577-2.04 3.592 0 5.577 1.893 5.577 5.062v4.984c0 .433.135.564.499.564h.783Zm-4.873-2.303v-.41H88.78c-1.654 0-2.358.51-2.358 1.539 0 .996.68 1.56 2.017 1.56 1.836 0 3.015-1.073 3.015-2.689Z"
              fill="#012D26"
            />
          </svg>
          <nav
            class="menu"
          >
            <a
              class="menuItem"
              href="/#"
            >
              Cundinamarca
            </a>
            <a
              class="menuItem"
              href="/#"
            >
              Antioquia
            </a>
            <a
              class="menuItem"
              href="/#"
            >
              Recursos
            </a>
          </nav>
          <button
            class="menuItem profileItem"
            data-testid="profile-button"
            type="button"
          >
            Mi Perfil
            <svg
              class="transform dropdownIcon"
              fill="none"
              height="9"
              width="14"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="m.62 1.63.884-.884L7 6.24 12.495.746l.884.883L7 8.01.62 1.629Z"
                fill="#012D26"
                fill-rule="evenodd"
              />
            </svg>
            <span
              class="rounded-full w-3 h-3 absolute top-0 right-0 bg-accent notificationBadge"
              data-testid="notification-badge"
            />
          </button>
        </div>
      </div>
    </div>
  </header>
  <p>
    Hello World
  </p>
</div>
`;

exports[`layouts/AppShell it renders without any extra props 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="center"
    >
      <div
        class="content"
      >
        <div
          class="flex items-center"
        >
          <div
            class="menuIconContainer"
          >
            <svg
              class="menuIcon"
              fill="none"
              height="32"
              title="Menu"
              width="32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M22 25H2v2h20v-2ZM30 15H2v2h28v-2ZM30 5H2v2h28V5Z"
                fill="#5B6866"
              />
            </svg>
            <span
              class="rounded-full w-3 h-3 absolute top-0 right-0 bg-accent notificationBadge"
              data-testid="notification-badge"
            />
          </div>
          <svg
            class="logo hidden md:inline-block"
            fill="none"
            height="25"
            title="La Haus logo"
            width="125"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M13.046.678v9.598a.69.69 0 0 1-.696.68h-1.393a.69.69 0 0 1-.696-.68V3.754c0-.487-.506-.82-.965-.634L.436 6.675A.69.69 0 0 0 0 7.309V22.98c0 .378.316.68.696.68h8.86a.69.69 0 0 0 .697-.68v-5.99a.69.69 0 0 1 .696-.68h1.393a.69.69 0 0 1 .696.68v5.99c0 .378.316.68.696.68h8.86a.69.69 0 0 0 .697-.68V4.24a.69.69 0 0 0-.435-.634L13.995.052a.69.69 0 0 0-.95.626Z"
              fill="#3ECFAF"
            />
            <path
              d="M47.365 20.677v2.403a.557.557 0 0 1-.562.549H35.965a.557.557 0 0 1-.562-.55V6.273c0-.302.253-.549.562-.549h2.753c.308 0 .561.247.561.549V20.12h7.524a.564.564 0 0 1 .562.556ZM62.364 20.841v2.303a.49.49 0 0 1-.498.487h-1.337c-1.472 0-2.27-.48-2.508-1.592-.024-.108-.174-.139-.238-.046-.712 1.167-2.183 1.916-4.192 1.916-2.935 0-4.795-1.584-4.795-4.041 0-2.504 1.939-4.042 5.657-4.042h3.038v-.386c0-1.43-.736-2.095-2.279-2.095a4.769 4.769 0 0 0-3.093 1.128c-.206.17-.514.178-.704-.015l-1.48-1.522a.473.473 0 0 1-.007-.665c1.305-1.283 3.306-2.04 5.577-2.04 3.592 0 5.578 1.893 5.578 5.062v4.984c0 .433.134.564.498.564h.783Zm-4.873-2.303v-.41h-2.666c-1.654 0-2.358.51-2.358 1.539 0 .996.68 1.56 2.017 1.56 1.828 0 3.007-1.073 3.007-2.689ZM80.3 6.272V23.08a.557.557 0 0 1-.561.549h-2.73a.557.557 0 0 1-.561-.549v-6.87h-7.556a.557.557 0 0 0-.561.549v6.321a.557.557 0 0 1-.562.549h-2.753a.557.557 0 0 1-.562-.549V6.272c0-.301.253-.548.562-.548h2.753c.308 0 .562.247.562.548v6.46h7.555a.557.557 0 0 0 .562-.548V6.272c0-.301.253-.548.561-.548h2.73c.316 0 .561.247.561.548ZM110.541 11.081v12.055c0 .27-.23.495-.507.495h-2.626a.504.504 0 0 1-.506-.495v-1.421c-.736 1.383-2.334 2.28-4.296 2.28-2.8 0-4.375-1.763-4.375-4.862v-8.052c0-.27.23-.495.506-.495h2.706c.277 0 .506.224.506.495v7.102c0 1.684.554 2.504 2.097 2.504 1.701 0 2.777-.974 2.777-2.736v-6.87c0-.27.229-.495.506-.495h2.705a.5.5 0 0 1 .507.495ZM112.956 21.297l1.179-1.553a.548.548 0 0 1 .768-.108c1.06.819 2.531 1.337 3.987 1.337 1.313 0 2.017-.433 2.017-1.098 0-.564-.474-.92-1.598-1.174l-2.223-.51c-2.8-.665-3.9-1.948-3.9-3.733 0-2.48 2.096-4.22 5.633-4.22 2.12 0 3.987.627 5.221 1.647a.533.533 0 0 1 .095.734l-1.202 1.6a.557.557 0 0 1-.752.116c-1.139-.735-2.294-1.052-3.41-1.052-1.281 0-1.883.41-1.883 1.052 0 .463.396.795 1.575 1.05l2.302.534c2.856.664 3.876 1.947 3.876 3.763 0 2.635-2.12 4.32-5.759 4.32-2.421 0-4.525-.742-5.862-2.002a.509.509 0 0 1-.064-.703ZM96.327 20.841v2.303a.49.49 0 0 1-.498.487h-1.337c-1.472 0-2.27-.48-2.508-1.592-.024-.108-.174-.139-.238-.046-.712 1.167-2.183 1.916-4.193 1.916-2.935 0-4.794-1.584-4.794-4.041 0-2.504 1.939-4.042 5.657-4.042h3.038v-.386c0-1.43-.736-2.095-2.279-2.095a4.769 4.769 0 0 0-3.093 1.128c-.206.17-.514.178-.704-.015l-1.48-1.522a.473.473 0 0 1-.007-.665c1.305-1.283 3.306-2.04 5.577-2.04 3.592 0 5.577 1.893 5.577 5.062v4.984c0 .433.135.564.499.564h.783Zm-4.873-2.303v-.41H88.78c-1.654 0-2.358.51-2.358 1.539 0 .996.68 1.56 2.017 1.56 1.836 0 3.015-1.073 3.015-2.689Z"
              fill="#012D26"
            />
          </svg>
          <svg
            class="logo md:hidden"
            fill="none"
            height="32"
            title="La Haus logo"
            width="31"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M17.11.917v12.981c0 .512-.416.92-.914.92H14.37a.918.918 0 0 1-.913-.92V5.077a.919.919 0 0 0-1.266-.857L.571 9.028A.934.934 0 0 0 0 9.885V31.08c0 .512.415.92.913.92h11.62a.918.918 0 0 0 .914-.92v-8.1c0-.512.415-.92.913-.92h1.826c.508 0 .913.419.913.92v8.1c0 .512.415.92.913.92h11.62a.918.918 0 0 0 .913-.92V5.735a.934.934 0 0 0-.57-.857L18.355.071c-.582-.251-1.246.198-1.246.846Z"
              fill="#3ECFAF"
            />
          </svg>
        </div>
        <div
          class="backdrop"
        />
        <button
          aria-label="Cerrar Menú"
          class="close hidden"
          type="button"
        >
          <span
            aria-hidden="true"
            class="closeIcon"
          >
            ×
          </span>
        </button>
        <div
          class="transform nav"
        >
          <svg
            class="logo"
            fill="none"
            height="25"
            title="La haus logo"
            width="125"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M13.046.678v9.598a.69.69 0 0 1-.696.68h-1.393a.69.69 0 0 1-.696-.68V3.754c0-.487-.506-.82-.965-.634L.436 6.675A.69.69 0 0 0 0 7.309V22.98c0 .378.316.68.696.68h8.86a.69.69 0 0 0 .697-.68v-5.99a.69.69 0 0 1 .696-.68h1.393a.69.69 0 0 1 .696.68v5.99c0 .378.316.68.696.68h8.86a.69.69 0 0 0 .697-.68V4.24a.69.69 0 0 0-.435-.634L13.995.052a.69.69 0 0 0-.95.626Z"
              fill="#3ECFAF"
            />
            <path
              d="M47.365 20.677v2.403a.557.557 0 0 1-.562.549H35.965a.557.557 0 0 1-.562-.55V6.273c0-.302.253-.549.562-.549h2.753c.308 0 .561.247.561.549V20.12h7.524a.564.564 0 0 1 .562.556ZM62.364 20.841v2.303a.49.49 0 0 1-.498.487h-1.337c-1.472 0-2.27-.48-2.508-1.592-.024-.108-.174-.139-.238-.046-.712 1.167-2.183 1.916-4.192 1.916-2.935 0-4.795-1.584-4.795-4.041 0-2.504 1.939-4.042 5.657-4.042h3.038v-.386c0-1.43-.736-2.095-2.279-2.095a4.769 4.769 0 0 0-3.093 1.128c-.206.17-.514.178-.704-.015l-1.48-1.522a.473.473 0 0 1-.007-.665c1.305-1.283 3.306-2.04 5.577-2.04 3.592 0 5.578 1.893 5.578 5.062v4.984c0 .433.134.564.498.564h.783Zm-4.873-2.303v-.41h-2.666c-1.654 0-2.358.51-2.358 1.539 0 .996.68 1.56 2.017 1.56 1.828 0 3.007-1.073 3.007-2.689ZM80.3 6.272V23.08a.557.557 0 0 1-.561.549h-2.73a.557.557 0 0 1-.561-.549v-6.87h-7.556a.557.557 0 0 0-.561.549v6.321a.557.557 0 0 1-.562.549h-2.753a.557.557 0 0 1-.562-.549V6.272c0-.301.253-.548.562-.548h2.753c.308 0 .562.247.562.548v6.46h7.555a.557.557 0 0 0 .562-.548V6.272c0-.301.253-.548.561-.548h2.73c.316 0 .561.247.561.548ZM110.541 11.081v12.055c0 .27-.23.495-.507.495h-2.626a.504.504 0 0 1-.506-.495v-1.421c-.736 1.383-2.334 2.28-4.296 2.28-2.8 0-4.375-1.763-4.375-4.862v-8.052c0-.27.23-.495.506-.495h2.706c.277 0 .506.224.506.495v7.102c0 1.684.554 2.504 2.097 2.504 1.701 0 2.777-.974 2.777-2.736v-6.87c0-.27.229-.495.506-.495h2.705a.5.5 0 0 1 .507.495ZM112.956 21.297l1.179-1.553a.548.548 0 0 1 .768-.108c1.06.819 2.531 1.337 3.987 1.337 1.313 0 2.017-.433 2.017-1.098 0-.564-.474-.92-1.598-1.174l-2.223-.51c-2.8-.665-3.9-1.948-3.9-3.733 0-2.48 2.096-4.22 5.633-4.22 2.12 0 3.987.627 5.221 1.647a.533.533 0 0 1 .095.734l-1.202 1.6a.557.557 0 0 1-.752.116c-1.139-.735-2.294-1.052-3.41-1.052-1.281 0-1.883.41-1.883 1.052 0 .463.396.795 1.575 1.05l2.302.534c2.856.664 3.876 1.947 3.876 3.763 0 2.635-2.12 4.32-5.759 4.32-2.421 0-4.525-.742-5.862-2.002a.509.509 0 0 1-.064-.703ZM96.327 20.841v2.303a.49.49 0 0 1-.498.487h-1.337c-1.472 0-2.27-.48-2.508-1.592-.024-.108-.174-.139-.238-.046-.712 1.167-2.183 1.916-4.193 1.916-2.935 0-4.794-1.584-4.794-4.041 0-2.504 1.939-4.042 5.657-4.042h3.038v-.386c0-1.43-.736-2.095-2.279-2.095a4.769 4.769 0 0 0-3.093 1.128c-.206.17-.514.178-.704-.015l-1.48-1.522a.473.473 0 0 1-.007-.665c1.305-1.283 3.306-2.04 5.577-2.04 3.592 0 5.577 1.893 5.577 5.062v4.984c0 .433.135.564.499.564h.783Zm-4.873-2.303v-.41H88.78c-1.654 0-2.358.51-2.358 1.539 0 .996.68 1.56 2.017 1.56 1.836 0 3.015-1.073 3.015-2.689Z"
              fill="#012D26"
            />
          </svg>
          <nav
            class="menu"
          >
            <a
              class="menuItem"
              href="/#"
            >
              Cundinamarca
            </a>
            <a
              class="menuItem"
              href="/#"
            >
              Antioquia
            </a>
            <a
              class="menuItem"
              href="/#"
            >
              Recursos
            </a>
          </nav>
          <button
            class="menuItem profileItem"
            data-testid="profile-button"
            type="button"
          >
            Mi Perfil
            <svg
              class="transform dropdownIcon"
              fill="none"
              height="9"
              width="14"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="m.62 1.63.884-.884L7 6.24 12.495.746l.884.883L7 8.01.62 1.629Z"
                fill="#012D26"
                fill-rule="evenodd"
              />
            </svg>
            <span
              class="rounded-full w-3 h-3 absolute top-0 right-0 bg-accent notificationBadge"
              data-testid="notification-badge"
            />
          </button>
        </div>
      </div>
    </div>
  </header>
</div>
`;
